src/components/news.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <q-carousel class="component-top" style="max-width: 700px;margin:auto; height:300px">
+    <q-carousel-slide :img-src="item.img">
+      <div class="absolute-top-left custom-caption-top">
+        <div class="subtitle text-white q-ml-sm q-mt-sm" >{{item.sub}}</div>
+      <div class="q-display-1 text-white q-ml-sm q-mb-sm" >{{item.title}}</div>
+        <q-btn class=" fixed-bottom-left q-ml-sm q-mb-sm" :class="{ hidden: item.isBtnHidden }" color="white" text-color="brown"  rounded>
+          <q-item  @click.native="show">
```

Comment:
what is the purpose of .native ?

Reply:
when we use <q-item> must add  (native)
but this is redundant. i removed it